go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	github.com/hashicorp/terraform v0.11.13
 	github.com/vmware/go-vcloud-director/v2 v2.2.0-alpha.1
 )
+
+replace github.com/vmware/go-vcloud-director/v2 => github.com/Didainius/go-vcloud-director/v2 v2.2.0-alpha.1.0.20190417085158-6cbca97d9053
```

Comment:
What about merging the govcd PR, bumping up govcd to v2.2.0-alpha.2 and then dropping this replace from this PR?

Reply:
If we do that, we always need to review PRs separately. The "replace" method, which is made for this purpose, allows developers to submit the whole code for review at once.
I suggest we leave it in place

Reply:
tldr; This is done only to make TravisCI happy.

This is probably a wider discussion how we handle PRs when they depend on govcd changes. 
This setting allows to see if Travis CI is ok or not (and in future hopefully test suites) without having a PR merged into govcd.

However the drawback as you just spotted is that there's a lot of changed files (rebuilt vendor directory) and constant madness with go.mod and go.sum.

Before real merge go.mod **must** be reset to newly tagged govcd commit as I've stated it in the PR message:
> 
> **Note**. go.mod is temporary pointed to my fork to make Travis work. It will be changed before merge.